src/app/pages/system/tunable/tunable-form/tunable-form.component.ts

```diff
@@ -36,50 +36,60 @@ export class TunableFormComponent {
       type: 'input',
       name: 'tun_var',
       placeholder: T('Variable'),
-      tooltip: T('The name of the sysctl or driver to load.'),
+      tooltip: T('Enter the name of the loader, sysctl, or rc.conf\
+                  variable to configure.<br>\
+                  <b>loader</b>: Tunables that can be set only at boot\
+                  and not later.<br>\
+                  <b>rc.conf</b>: Enable or disable system services and\
+                  daemons.<br>\
+                  <b>sysctl</b>: Tunables that can be set anytime.'),
       required: true,
       validation : [ Validators.required ]
     },
     {
       type: 'textarea',
       name: 'tun_value',
       placeholder: T('Value'),
-      tooltip: T('Set a value for the variable. Refer to the man page\
-                  for the specific driver or the <a\
-                  href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/"\
-                  target="_blank">FreeBSD Handbook</a> for suggested\
-                  values.'),
+      tooltip: T('Enter a value to use for the <a\
+                  href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/boot-introduction.html#boot-loader-commands"\
+                  target="_blank">loader</a>, <a\
+                  href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/configtuning-sysctl.html"\
+                  target="_blank">sysctl</a>, or <a\
+                  href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/config-tuning.html"\
+                  target="_blank">rc.conf</a> variable.'),
       required: true,
       validation : [ Validators.required ]
     },
     {
       type: 'select',
       name: 'tun_type',
       placeholder: T('Type'),
-      tooltip: T('Creating or editing a <i>Sysctl</i> immediately\
-                  updates the <b>Variable</b>. A reboot is required when\
-                  a <i>Loader</i> or <i>rc.conf</i> value changes.\
-                  A tunable remains at boot and across upgrades unless\
-                  it is deleted or <b>Enabled</b> is unset.'),
+      tooltip: T('Creating or editing a <i>sysctl</i> immediately\
+                  updates the <b>Variable</b> to the configured\
+                  <b>Value</b>. When using a <i>loader</i> or\
```

Comment:
Simplify and avoid pauses: "A restart is required to apply <i>loader</i> or <i>rc.conf</i> tunables. Configured tunables remain in effect until deleted or <i>Enabled</i> is unset."

Reply:
done.
Removed the bold and the italics from Enabled. Filed names are identified with capitalization, no italics needed.